pkg/core/variationDecider_test.go

```diff
@@ -97,38 +97,38 @@ func TestFindTargetedVariation(t *testing.T) {
 	vwoInstance := getInstanceWithoutStorage("./testdata/testVariation.json")
 	options := schema.Options{
 		VariationTargetingVariables: map[string]interface{}{"_vwo_user_id": "USER_1"},
-		RevenueGoal:     12,
+		RevenueValue:     12,
 	}
 
 	userID := "USER_1"
 	campaign := vwoInstance.SettingsFile.Campaigns[2]
-	actual, _ := FindTargetedVariation(vwoInstance, userID, campaign, options)
+	actual, _, _ := FindTargetedVariation(vwoInstance, userID, campaign, options)
```

Comment:
do not ignore values in the test file. write an assert for the same.

Reply:
Ok